docker/settings.yml

```diff
@@ -1,27 +1,36 @@
-folderPath: /opt/takipi-storage/storage
-maxUsedStoragePercentage: 0.95
 enableCors: true
 corsOrigins: "*"
 
+s3Fs:
+    bucket: <bucket>
+    credentials:
+      accessKey: <key> 
+      secretKey: <secret> 
+
 server:
   applicationConnectors:
     - type: http
       port: 8080
+# this requires the alpn-boot library on the JVM's boot classpath
```

Comment:
it's time we remove this spdy nonesense and change it to a https 8443 commented out